specs/lib/removeUnusedElementsSpec.js

```diff
@@ -391,6 +416,18 @@ describe('removeUnusedElements', () => {
         });
     });
 
+    it('correctly removes/keeps textures', () => {
+        expect(gltf.textures.length).toBe(1);
+    });
+
+    it('correctly removes/keeps samplers', () => {
+        expect(gltf.samplers.length).toBe(1);
+    });
+
+    it('correctly removes/keeps images', () => {
+        expect(gltf.samplers.length).toBe(1);
+    });
```

Comment:
These tests could be improved.

* Incorporate these tests into https://github.com/AnalyticalGraphicsInc/gltf-pipeline/blob/master/specs/lib/removeUnusedElementsSpec.js#L377-L378 rather than being separate tests if possible
* `correctly removes/keeps textures` does not check for unused texture removal. Add two textures to the json, one used and one unused, and then check that the length is 1.
* `correctly removes/keeps images` is checking samplers instead of images. Like above, add an unused image to the json.